client/src/components/CustomTables.js

```diff
@@ -173,6 +174,28 @@ const upcomingOrOngoingTableStyles = makeStyles({
 export function UpcomingOrOngoingTable(props) {
   const classes = upcomingOrOngoingTableStyles();
   const [ongoingInterviewList, setOngoingInterviewList] = useState([]);
+  const [newongoingInterviewList, setnewongoingInterviewList] = useState([]);
+  function cancelInterview(id) {
+    console.log(id);
+    fetch("interviews/remove", {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ id: id }),
+    })
+      .then((res) => console.log(res))
+      .then(() => {
+        let arr = ongoingInterviewList;
+        let index = arr.indexOf(id);
+        console.log(index);
+        if (index !== -1) {
+          console.log(ongoingInterviewList);
+          arr.splice(index, 1);
```

Comment:
With state, you shouldn't be modifying the array directly, you should be creating a copy 
So that means you can use slice instead